hera_cal/abscal.py

```diff
@@ -1355,6 +1369,11 @@ def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose
         gain_pols = AC.gain_pols
         gain_keys = gain_dict.keys()
 
+        # join gain_flag_dict with flag_dict
+        for k in flag_dict:
+            if k in gain_flag_dict:
+                flag_dict[k] += gain_flag_dict[k]
```

Comment:
What's the logic here? Were we missing flags before?

Reply:
the flags were always False, so I'm just propagating the solar flags